CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ The following is a curated list of changes in the Enact agate module, newest cha
 - `agate/ArcSlider` style to match latest design for Silicon skin
 - `agate/Button` badge background color for Cobalt and Copper skins
 - `agate/ContextualPopupDecorator` style to match latest design for Silicon skin
+- `agate/Input` to position correctly iconBefore and iconAfter on Carbon, Cobalt, Copper, Electro, Titanium skins
```

Comment:
I think this PR is too old.. Let's do merge develop.

Reply:
```suggestion
- `agate/Input` to position correctly `iconBefore` and `iconAfter` on Carbon, Cobalt, Copper, Electro, and Titanium skins
```

Reply:
indeed. very old. The merge is done